packages/strapi-plugin-content-type-builder/admin/src/components/RelationBox/index.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InputTextWithErrors as InputText } from 'strapi-helper-plugin';
+
+import StyledRelationBox from './StyledRelationBox';
+import FeaturePicker from '../FeaturePicker';
+
+const RelationBox = ({
+  autoFocus,
+  didCheckErrors,
+  errors,
+  featureName,
+  features,
+  main,
```

Comment:
`isMain` check out this [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md)